Zend/zend_builtin_functions.c

```diff
@@ -1009,7 +1009,7 @@ ZEND_FUNCTION(get_class)
 {
 	zval *obj = NULL;
 
-	if (zend_parse_parameters(ZEND_NUM_ARGS(), "|o!", &obj) == FAILURE) {
+	if (zend_parse_parameters(ZEND_NUM_ARGS(), "|o", &obj) == FAILURE) {
```

Comment:
shouldn't the logic also be removed from source which handles `obj == NULL` at `1016` line?